examples/r-api/README.md

```diff
@@ -0,0 +1,118 @@
+
+```R
+install.packages('plumber')
+install.packages("dplyr")
+```
+
+Now let us create another R script room.R, this script will contain our regression model, which we will wrap with plumber. In the following code in first line we have placed oxigen2-like comments for describing what this API will do. Now we will define operation and path . Here we have used `GET` operation (read data) and path is `/predict`. This endpoint will return the predicted price of a house.
+
+This function is taking 3 arguments: `response`, `bedrooms` which represents number of bedrooms and `year` which represents year build. `bedrooms` and `year` are inputs from the client.
+
+Inside function we read houseprice data and perform linear regression to build a model. We then predict price of house for given inputs. We are returning HTTP response 400 if parameters passed are not in range of training data. If the parameters fall within valid range, API will return predicted house price.
+
+```R
+#* Predict price of a house
+#* @get /predict
+function(bedrooms,year,res){
+  data <- read.csv("https://saturn-public-data.s3.us-east-2.amazonaws.com/examples/dashboard/housePriceData.csv")
+  l = lm(SalePrice~BedroomAbvGr+YearBuilt, data)
+  bedrooms<-as.numeric(bedrooms)
+  year<-as.numeric(year)
+  if(between(bedrooms,0, 8) & between(year, 1871,2100)){
+    X <- data.frame(BedroomAbvGr=bedrooms,YearBuilt=year)
+    return(predict(l, newdata = X))
+    
+  } else {
+    res$status <- 400  
+    return(list(error = "Please enter BedroomAbvGr between 0 and 8. Enter YearBuilt between 1872 and 2100."))
+  }  
+}
+```
+Now create another R script which will run above piece of code. Let's name this as runapi.R.  Load the plumber package. Start the server using plumber object. Since the host listens on 0.0.0.0, it will be reachable on an appropriate interface address to the connection. Port 8000 is a common HTTP port for web servers.
+```R
+library(dplyr)
+library(plumber)
+plumber::pr_run(plumber::plumb("room.R"), port=8000, host="0.0.0.0")
+```
+
+## Deployment
+
+ Save all these files (room.R, install-packages.R,runapi.R and startup-script.sh) in a Github repo. Now click **New Deployment**. It can be found on top right side of the resource page. 
+
+![deploy](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/plumber_deployment.png "doc-image")
+
+
+To run, add following to command field.
+```bash
+Rscript runapi.R
+```
+Add path to file in working directory field as shown below. In the screenshot below, my file runapi.R is inside repository Dashboard. If you aren't familiar with how to set up SSH credentials and add git repositories to Saturn Cloud check [here](https://saturncloud.io/docs/using-saturn-cloud/gitrepo/).
+
+![deploy](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/working-directory.png "doc-image")
+
+
+Go to Advanced Settings -> Start Script.  Add following :
+
+sh /home/jovyan/git-repos/Dashboard/startup-script.sh
```

Comment:
GitHub won't let me edit this part directly so change it to have ```R with:

```
Rscript setup.R
```